### PR TITLE
Add section for test suite and implementation report

### LINKED ISSUE #20: Add section pointing to the test suite and implementation report.
We need a section that links to the test suite and also that allows developers to judge which existing implementation best fits their needs. https://w3c.github.io/vc-test-suite/implementations/

### PR DESCRIPTION
**Pull request by deiu**
There was no pointer to the test suite, nor to the implementation report.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1489,5 +1489,24 @@ <h3>Further Techniques</h3>
...
+    <section>
+      <h2>Test suite</h2>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+        The W3C Verifiable Claims Working Group has produced a 
+        <a href="https://github.com/w3c/vc-test-suite/">test suite</a> in order 
+        for implementers to validate their conformance with the current specifications. 
+        Any conforming implementation MUST pass all tests in the test suite.
```
Comment (brentzundel) [inline index.html]: Since this section is non-normative, it shouldn't have a statement with MUST.

**Review by brentzundel — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html]: Perhaps `Conforming implementations are expected to pass all...` 

Except that many tests apply to optional features, so some further language massage is needed.

Comment (deiu) [reply index.html]: Good points, I've updated the text accordingly. Thanks!

File: index.html [hunk 2]
```diff
@@ -1499,7 +1499,7 @@ <h2>Test suite</h2>
         The W3C Verifiable Claims Working Group has produced a 
         <a href="https://github.com/w3c/vc-test-suite/">test suite</a> in order 
         for implementers to validate their conformance with the current specifications. 
-        Any conforming implementation MUST pass all tests in the test suite.
+        Conforming implementation are encouraged to pass all tests in the test suite.
```
Comment (TallTed) [inline index.html]: `Conforming implementation are`
-> `Conforming implementations are`

Comment (deiu) [reply index.html]: Grrr...it's late. Fixed!

File: index.html [hunk 3]
```diff
@@ -1489,5 +1489,24 @@ <h3>Further Techniques</h3>
...
+    <section>
+      <h2>Test suite</h2>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+        The W3C Verifiable Claims Working Group has produced a 
+        <a href="https://github.com/w3c/vc-test-suite/">test suite</a> in order 
+        for implementers to validate their conformance with the current specifications. 
+        Conforming implementations are encouraged to pass all tests in the test suite.
```
Comment (brentzundel) [inline index.html]: It may be better with this line removed entirely: "Conforming implementations are encouraged to pass all tests in the test suite."

Comment (TallTed) [reply index.html]: I did say that more language was needed...

Comment (deiu) [reply index.html]: I have removed the entire line until we come up with a better text.

File: index.html [hunk 4]
```diff
@@ -1498,8 +1498,7 @@ <h2>Test suite</h2>
       <p>
         The W3C Verifiable Claims Working Group has produced a 
         <a href="https://github.com/w3c/vc-test-suite/">test suite</a> in order 
-        for implementers to validate their conformance with the current specifications. 
-        Conforming implementations are encouraged to pass all tests in the test suite.
+        for implementers to validate their conformance with the current specifications.
```
Comment (TallTed) [inline index.html]: `validate`
-> `check` or `test` or `confirm`

**Review by brentzundel — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Pull request merged**